service/src/main/kotlin/com/mashup/dojo/service/MemberService.kt

```diff
@@ -90,6 +92,26 @@ class DefaultMemberService(
         return MemberId(id)
     }
 
+    override fun findAllMember(): List<Member> {
+        return memberRepository.findAll().stream().map { m ->
+            val platform = MemberPlatform.findByValue(m.platform)
+            val gender = MemberGender.findByValue(m.gender)
+            val imageId = m.profileImageId?.let { profileImageId -> ImageId(profileImageId) }
+            Member.convertToMember(
+                m.id,
+                m.fullName,
+                m.secondInitialName,
+                imageId,
+                m.ordinal,
+                platform,
+                gender,
+                m.point,
+                m.createdAt,
+                m.updatedAt
+            )
+        }.toList()
+    }
```

Comment:
```suggestion
    override fun findAllMember(): List<Member> {
        return memberRepository.findAll()
            .map { m ->
                val platform = MemberPlatform.findByValue(m.platform)
                val gender = MemberGender.findByValue(m.gender)
                val imageId = m.profileImageId?.let { ImageId(it) }
                
                Member.convertToMember(
                    id = m.id,
                    fullName = m.fullName,
                    secondInitialName = m.secondInitialName,
                    profileImageId = imageId,
                    ordinal = m.ordinal,
                    platform = platform,
                    gender = gender,
                    point = m.point,
                    createdAt = m.createdAt,
                    updatedAt = m.updatedAt
                )
            }
    }
```
stream() & toList() 로 안하고 map{} 으로만 처리해도 동일하게 처리할 수 있을 것 같네요

Reply:
아하 코틀린에선 이렇게 가능하군요. 감사합니다~ 반영하겠습니다!